static/redfish/v1/schema/OemEthernetInterface_v1.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What schema did you start with for this? 
This doesn't match the new style Ed introduced upstream: 
https://github.com/openbmc/bmcweb/commit/5b5574ace856c7f787528ab8f5e5352a90f2a6cb

Reply:
Updated schemas as per new style introduced.
and generated json schemas from xml using https://github.com/DMTF/Redfish-Tools